src/Calypso-SystemTools-Core/ClyBrowserMorph.extension.st

```diff
@@ -26,20 +26,38 @@ ClyBrowserMorph >> confirmEmptySystemQuery: aQuery [
 	^false
 ]
 
+{ #category : #'*Calypso-SystemTools-Core' }
+ClyBrowserMorph >> confirmEmptySystemQuery: aQuery excluding: classes [
+	
+	| showResult result |
+	
+	result := aQuery execute.
+	(result isEmpty or: [ ((result items collect: [:each | each origin]) difference: classes ) isEmpty]) ifTrue: [ ^true ].
+	
+	showResult := UIManager default confirm: 'There are '
+		, result size asString, ' ', aQuery description, '. Show them?'.
+	showResult ifTrue: [self spawnQueryBrowserOn: aQuery].
```

Comment:
I think we need to show only external references instead of all of them. And for this we can introduce ClyExternalClassReferencesQuery. In that case this method will not be needed.

Otherwise the method needs to be renamed at least because system query (what is in method name) is not only about methods. Queries can return other objects (classes, comments)